qa/cross_chain_evm_to_be_deleted.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import time
+from decimal import Decimal
+from eth_utils import remove_0x_prefix
+import os
+
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from SidechainTestFramework.account.ac_utils import format_evm, format_eoa
+from SidechainTestFramework.account.httpCalls.transaction.createEIP1559Transaction import createEIP1559Transaction
+from SidechainTestFramework.account.httpCalls.transaction.sendTransaction import sendTransaction
+from SidechainTestFramework.account.httpCalls.transaction.signTransaction import signTransaction
+from SidechainTestFramework.account.httpCalls.transaction.simpleapp.redeemVoteMessage import redeemVoteMessage
+from SidechainTestFramework.account.httpCalls.transaction.simpleapp.sendVoteMessage import \
+    sendVoteMessage
+from SidechainTestFramework.sc_boostrap_info import KEY_ROTATION_CIRCUIT
+from SidechainTestFramework.scutil import generate_next_block
+from httpCalls.sc2sc.createAccountRedeemMessage import createAccountRedeemMessage
+from test_framework.util import forward_transfer_to_sidechain, assert_true
+
+
+# THIS TEST WILL BE REMOVED WHEN THE STF WILL SUPPORT MULTIPLE SIDECHAINS AT ONCE
+class CrossChainEvmToBeDeleted(AccountChainSetup):
+    def __init__(self):
+        super().__init__(
+            withdrawalEpochLength=10,
+            block_timestamp_rewind=720 * 1200,
+            circuittype_override=KEY_ROTATION_CIRCUIT,
+            sc2sc_proving_key_file_name="proving",
+            sc2sc_verification_key_file_name="verification"
+        )
+
+    def do_send_raw_tx(self, raw_tx, evm_signer_address):
+        sc_node = self.sc_nodes[0]
+        signed_raw_tx = signTransaction(sc_node, fromAddress=evm_signer_address, payload=raw_tx)
+
+        tx_hash = sendTransaction(sc_node, payload=signed_raw_tx)
+        return tx_hash
+
```

Comment:
Consider adding:

`def sc_setup_chain(self):`
`       # mandatory for this test`
`       self.options.nonceasing = True`
`       super().sc_setup_chain()`


for avoiding to specify this mandatory setting via the `--nonceasing` flag

Reply:
thanks, done :+1: